packages/wrangler/e2e/deployments.test.ts

```diff
@@ -1107,3 +918,45 @@ Current Version ID: 00000000-0000-0000-0000-000000000000`);
 		);
 	});
 });
+
+function validateAssetUploadLogs(
+	output: { stdout: string },
+	{
+		includeDebug = false,
+		files = ["/404.html", "/index.html", "/[boop].html"],
```

Comment:
WET!

I think the default value should be moved to the caller to avoid having to inspect this function to understand what the test is actually doing

Reply:
Done